jaxley/modules/base.py

```diff
@@ -413,6 +470,8 @@ def _at_nodes(self, key: str, idx: Any) -> View:
 
         Keys can be `cell`, `branch`, `comp` and determine which index is used to filter.
         """
+        base_name = self.base.__class__.__name__
+        assert self.base._has_childview(key), f"{base_name} does not support {key}."
```

Comment:
Disallow stuff like `comp.cell(0)`. While I think there is technically no harm in allowing it  (the cell_index also shows up in `nodes`), it makes little sense and could lead to more problems than it solves for the user, i.e. them calling `comp.cell(1)`, which does not exist.

Reply:
great! Yes, let's disallow it.